.github/workflows/e2e-tests.yaml

```
@@ -4,6 +4,11 @@ on:
   push:
     branches:
       - '**'
+    paths-ignore:
+      - 'docs/**'
+      - '*.md'
+      - 'mkdocs.yml'
+      - 'requirements.txt'  # MkDocs requirements
```

Same story, if you want to use `paths` settings:

```suggestion
  pull_request:
    branches:
      - main
    paths-ignore:
      - 'docs/**'
      - '*.md'
      - 'mkdocs.yml'
      - 'requirements.txt'  # MkDocs requirements
```

---

This one also worked as I was expecting, the tests didn't run when I pushed charges to docs folder.

---

I changed to `pull_request` but kept the `**`